docs/installation.md

```diff
@@ -57,11 +57,22 @@ vagrant destroy
 
 ### Quick Setup 2: Docker
 SMACK can also be run in a [Docker][] container. We tested the Dockerfile on
-the following configurations:
+the following configuration:
```

Comment:
This should be plural, no? Since you do have two configurations.

Reply:
Done.